### PR TITLE
allow selection of catalog via --field for PSF

### LINKED ISSUE #104: --field argument has no effect in psf stage
In the `-s psf` stage `args.catalogue` is passed to the `myloopdef.run_psf` 
https://github.com/LCOGT/lcogtsnpipe/blob/021167830f25597d9ef19be808b399beabd4daee/trunk/bin/lscloop.py#L186-L189 

but in `lscloop.py`, `args.catalogue` or any similar variable (`catalogue`) is not updated to reflect any effect of `--field` kwarg.

So, adding `--field apass` does not actually use the apass catalogue for psf star selection and uses the default gaia catalogue.

In a later part of lscloop.py this is handled in the following way - 
https://github.com/LCOGT/lcogtsnpipe/blob/021167830f25597d9ef19be808b399beabd4daee/trunk/bin/lscloop.py#L229-L238

### PR DESCRIPTION
**Pull request by griffin-h**
Resolves #104

### CODE REVIEW

**Comment by griffin-h** [comment]
@navii98 I can't officially add you as a reviewer, but please check whether this resolves your issue.

**Comment by griffin-h** [comment]
I'll note that this has to happen inside `lscpsf.py` instead of `lscloop.py` because the catalog can be different for different images, e.g., in the case of running on standard fields.

File: trunk/src/lsc/myloopdef.py [hunk 1]
```diff
@@ -336,6 +336,10 @@ def run_psf(imglist, treshold=5, interactive=False, _fwhm='', show=False, redo=F
             cc=' --catalog '+catalog+' '
         else:
             cc=' '
+        if field is not None:
+            field_cmd = ' --field ' + field + ' '
+        else:
+            field_cmd = ' '
```
Comment (navii98) [inline trunk/src/lsc/myloopdef.py:342]: This creates an issue while running the `lscpsf.py` command.
In `lscloop.py` `--field` is always initialized with an empty string. So, the if statement here will always run.

**Comment by navii98** [comment]
Other than that, this pull request works as intended.
Both #104 and #105 resolved.

**Pull request merged**